Automattic/wp-calypso
# Stats: add feature `paid-stats` to Complete plan

## Proposed Changes

Add feature `stats-paid` to Complete plans for feature checking. 

Backend feature definition:

fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Sjcpbz%2Qsrngherf%2Spynff%2Qjcpbz%2Qsrngherf.cuc%3Se%3Qns33so56%23367-og

## Testing Instructions

* Open Calypso Live branch
* Open `/stats/day/:siteSlug`
* Ensure you see upgrade notice for a site without any paid Stats products
* Purchase a complete plan for the site
* Ensure the upgrade notice is not shown

## Pre-merge Checklist
